setup.py

```diff
@@ -169,7 +169,7 @@ def run(self):
         # Ensure our temporary build directory exists
         try:
             os.makedirs(build_temp)
-        except IOError:
+        except:
```

Comment:
Also: I made a port for FreeBSD. For some reason 'install' operation begins to build something. This normally doesn't happen with python ports. And python framework is very well behaved and stable in FreeBSD. So there is something strange with pynacl.